packages/react-accordion/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@fluentui/react-accordion",
+  "version": "9.0.0-alpha.0",
+  "description": "Fluent UI accordion component",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.1",
+    "@fluentui/react-conformance": "^0.2.2",
+    "@fluentui/scripts": "^1.0.0",
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/jest": "24.9.1",
+    "@types/react": "16.9.42",
+    "@types/react-dom": "16.9.10",
+    "@types/react-test-renderer": "^16.0.0",
+    "@types/webpack-env": "1.16.0",
+    "enzyme": "~3.10.0",
+    "enzyme-adapter-react-16": "^1.15.0",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6",
+    "react-test-renderer": "^16.3.0"
+  },
+  "dependencies": {
+    "@fluentui/react-context-selector": "^0.53.3",
+    "@fluentui/react-focus-management": "^9.0.0-alpha.5",
+    "@fluentui/react-make-styles": "^9.0.0-alpha.7",
+    "@fluentui/react-theme": "^9.0.0-alpha.5",
+    "@fluentui/react-theme-provider": "^9.0.0-alpha.7",
+    "@fluentui/react-utilities": "^9.0.0-alpha.3",
+    "@reach/descendants": "0.13.2",
```

Comment:
```suggestion
```

I don't think that we are using this dependency as it was copied into `descendants.tsx`.